### PR TITLE
Add production-based carbon emissions parameter

### PR DESCRIPTION
**Pull request by ReemaMohanty**
File name has been changed.

### CODE REVIEW

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+description: Carbon Emissions, Production of goods and emissions
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]:
```suggestion
description: Carbon emissions produced in the UK.
```

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  2017-01-01: 399_000_000
+  2018-01-01: 406_000_000
+  2019-01-01: 371_000_000
+  metadata:
+  unit: tonnes CO2
+  reference:
+    - title: BEIS 2019 Updated Energy & Emissions Projections
+      href: https://www.ons.gov.uk/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2021#greenhouse-gas-emissions
+
+
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: We can remove this extra newline.

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+description: Carbon Emissions, Production of goods and emissions
+values:
+  2016-01-01: 423_000_000
+  2017-01-01: 399_000_000
+  2018-01-01: 406_000_000
+  2019-01-01: 371_000_000
+  metadata:
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]:
```suggestion
metadata:
```

**Review by nikhilwoodruff — CHANGES_REQUESTED** [review]
Great, thanks @ReemaMohanty! Just a few comments, and could we rename the parameter to not include `(`, `)` or `&` (perhaps `production.yaml` is fine)? Then, just adding an entry to the changelog and bumping the last version number would be great.

**Comment by ReemaMohanty** [comment]
> Great, thanks @ReemaMohanty! Just a few comments, and could we rename the parameter to not include `(`, `)` or `&` (perhaps `production.yaml` is fine)? Then, just adding an entry to the changelog and bumping the last version number would be great.

Yeah. Production will be fine.

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml (hunk 1 above)
Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: As  it deals with  production of goods and services, I have provided the description in this manner.

Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: How about `Carbon emissions from the production of goods and services in the UK`? I think we can uncapitalise `Emissions` and `Production` also.

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: (I think it's still there btw, re-opening)

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ values:
   metadata:
   unit: tonnes CO2
   reference:
-    - title: BEIS 2019 Updated Energy & Emissions Projections
+      title: BEIS 2019 Updated Energy & Emissions Projections
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: I think this dash didn't need to be removed; it signifies this reference is one in a list of references.

File: openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml (hunk 1 above)
Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: Done

Comment (nikhilwoodruff) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: Thanks, just the uncapitalising to go I think.

Comment (ReemaMohanty) [reply openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml]: sure

**Review by nikhilwoodruff — COMMENTED** [review]
Could we also uncapitalise the filename? Generally we don't capitalise filenames or code variables (with the exception of Python class declarations).

**Comment by ReemaMohanty** [comment]
Sure.I will uncapitalise the file name.

File: openfisca_uk/parameters/consumption/carbon/production/Production.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  2017-01-01: 399_000_000
+  2018-01-01: 406_000_000
+  2019-01-01: 371_000_000
+  metadata:
+  unit: tonnes CO2
+  reference:
+      -title: BEIS 2019 Updated Energy & Emissions Projections
+      href: https://www.ons.gov.uk/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2021#greenhouse-gas-emissions
+
+
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/Production.yaml]: We can remove this line

File: openfisca_uk/parameters/consumption/carbon/production/Production.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+description: Carbon emissions from the production of goods and services in the UK
+values:
+  2016-01-01: 423_000_000
+  2017-01-01: 399_000_000
+  2018-01-01: 406_000_000
+  2019-01-01: 371_000_000
+  metadata:
+  unit: tonnes CO2
+  reference:
+      -title: BEIS 2019 Updated Energy & Emissions Projections
```
Comment (nikhilwoodruff) [inline openfisca_uk/parameters/consumption/carbon/production/Production.yaml]:
```suggestion
    - title: BEIS 2019 Updated Energy & Emissions Projections
```

**Review by nikhilwoodruff — APPROVED** [review]

**Comment by nikhilwoodruff** [comment]
Thanks @ReemaMohanty, merging!

**Comment by MaxGhenis** [comment]
Thanks @ReemaMohanty! 🚀

**Pull request merged**